00-HTML-CSS-basics/2-CSS -Basics - Selectors and properties/CSS/style.css

```diff
@@ -0,0 +1,44 @@
+header, footer
```

Comment:
Is just a thing of tastes but I think is more clearer and readable to write this as

```
header,
footer {
  background-color: #e2ffe6;
  text-align: center;
}
```